pike/smb2.py

```diff
@@ -1829,19 +1829,23 @@ class LeaseRequest(CreateRequestContext):
     # This class handles V2 requests as well.  Set
     # the lease_flags field to a non-None value
     # to enable the extended fields
-    def __init__(self, parent):
+    def __init__(self, parent, do_v2_lease):
         CreateRequestContext.__init__(self, parent)
         self.lease_key = array.array("B", [0] * 16)
         self.lease_state = 0
         # V2 fields
-        self.lease_flags = None
-        self.parent_lease_key = None
-        self.epoch = None
+        self.lease_flags = LeaseFlags.SMB2_LEASE_FLAG_NONE
```

Comment:
Also wondering if we should set `SMB2_LEASE_FLAG_PARENT_LEASE_KEY_SET` under condition.

Reply:
srv currently doesn't support parent lease keys, so we don't want to set the flag.  (If we set the flag, we'd also need to actually set a valid parent_lease_key -- which we can't because we don't support.   I believe parent_lease_key is related to directory leases.)